### ISSUE TITLE
Scale and Settings: Text content of 'Info status message' and 'Warning' message is not visible properly at high contrast black mode.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1056]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1245561

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.
Ease of access high contrast settings > Turn on High Contrast > High Contrast Black/High Contrast White

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section.
Create 'Sample Database'. Expand this section and then expand 'Persons' tree option.
Navigate through all the controls under SQL API Section and select "Scale & Settings".
Navigate to scale tab section controls and select 'Auto scale' radio button.
Enable high contrast black mode.
Verify the issue.

Actual Result:
Text content of 'Info status message' and 'Warning' message is not visible properly at high contrast black mode.

Expected Result:
Text content of 'Info status message' and 'Warning' message should be visible properly at high contrast black mode.


Note:
Similar issue does not exist under high contrast white mode.
This issue also exists for icons present under 'SQL API' section of the page.

![image](https://user-images.githubusercontent.com/8519407/132714900-aed45bd7-4289-400a-978e-a2364bb90f5b.png)

![image](https://user-images.githubusercontent.com/8519407/132714946-d9520f13-bf58-4a4a-aab0-bcbaa3e2590e.png)
